proto/buf/alpha/registry/v1alpha1/plugin_curation.proto

```diff
@@ -232,9 +235,10 @@ message RegistryConfig {
     MavenConfig maven_config = 3;
     SwiftConfig swift_config = 4;
     PythonConfig python_config = 5;
+    ArtifactConfig artifact_config = 6;
   }
   // Reserved for future remote registry types.
-  reserved 6 to 9;
+  reserved 7 to 9;
```

Comment:
Just a point of order: this is sort of an abuse of the `reserved` keyword, `reserved` is really meant to say "these were previously used", outside of hyper-optimization theres no need to reserve these fields ahead of time.

Reply:
You can still group them together in the source code, regardless of tag - tag is kind of irrelevant unless you're trying to save one byte (you're not)